cypress/utils/dashboards/saved_objects_management/commands.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+Cypress.Commands.add('selectDataSourceForImport', (dataSourceTitle) => {
+  cy.getElementByTestId('dataSourceSelectorComboBox').click();
+  cy.get('.euiFilterSelectItem')
+    .contains(dataSourceTitle) // Find the element containing dataSourceTitle
+    .click();
+});
+
+Cypress.Commands.add('deleteAllSavedObjects', () => {
+  cy.wait(1000);
+  cy.deleteSavedObjectByType('dashboard');
+  cy.wait(1000);
+  cy.deleteSavedObjectByType('index-pattern');
+  cy.wait(1000);
+  cy.deleteSavedObjectByType('search');
+  cy.wait(1000);
+  cy.deleteSavedObjectByType('visualization');
+  cy.wait(1000);
+  cy.deleteSavedObjectByType('metrics');
+  cy.wait(1000);
+  cy.deleteSavedObjectByType('vega');
+});
```

Comment:
metrics and vega are not saved object types, they are visualization types, let's also make it generic by  including all saved object types comes with OSD
![image](https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/32652829/6afc6ef2-8661-4f04-bf00-74356a1f3a37)


```suggestion
Cypress.Commands.add("deleteAllSavedObjects", () => {
  const types = [
    "index-pattern",
    "visualization",
    "dashboard",
    "search",
    "query",
    "url",
    "augment-vis",
    "homepage",
    "data-source",
    "visualization-visbuilder",
    "config",
  ];
  types.forEach((type) => {
    cy.deleteSavedObjectByType(type);
  });
});
```